shapely/geometry/base.py

```diff
@@ -351,8 +352,8 @@ def buffer(
         distance,
         resolution=16,
         quadsegs=None,
-        cap_style=CAP_STYLE.round,
-        join_style=JOIN_STYLE.round,
+        cap_style=BufferCapStyle.ROUND,
+        join_style=BufferJoinStyle.ROUND,
```

Comment:
Should geometry.base's buffer have references to the new enums or preserve the old ones?